src/Compilers/Core/Portable/Microsoft.CodeAnalysis.csproj

```diff
@@ -80,7 +80,6 @@
     <InternalsVisibleTo Include="Microsoft.CodeAnalysis.CSharp.ExpressionEvaluator.ExpressionCompiler.UnitTests" />
     <InternalsVisibleTo Include="Microsoft.CodeAnalysis.VisualBasic.ExpressionEvaluator.ExpressionCompiler.UnitTests" />
     <InternalsVisibleTo Include="Microsoft.CodeAnalysis.ExpressionEvaluator.ExpressionCompiler.Utilities" />
-    <InternalsVisibleTo Include="Microsoft.CodeAnalysis.ExternalAccess.RazorCompiler" />
```

Comment:
Note: we could keep those around longer just in case an older EA DLL gets in contact with newer CodeAnalysis DLL? I'm not sure.

Reply:
That can only happen in scenarios that are already unsupported. Let's just remove this.